docs/methods.md

```diff
@@ -1,155 +1,152 @@
+
+We're now ready to start sampling our data, and we'll do this in three steps. These subsampling steps are independent, so can be run in any order.
+
+First, we'll sample across time. By default, this will select seven genomes per seven day period. The file that gets generated as a result here isn't something that is very useful to view directly, but we'll use it in a few minutes to see how many sequences were retained at this step.
+
 ```
-$ grep -c "^>" unaligned-preprint-seqs.fasta
-466
+qiime genome-sampler sample-longitudinal --i-context-seqs filtered-context-seqs.qza --m-dates-file context-metadata.tsv --m-dates-column date --o-selection date-selection.qza
 ```
 
+Next, we'll sample across viral diversity. This will cluster sequences at a percent identity threshold of 99.95%, and select the centroid sequence from each cluster to include in downstream analyses.
+
 ```
-$ md5sum unaligned-preprint-seqs.fasta*
-6506af95df6c4c3b8671489418b6773f  unaligned-preprint-seqs.fasta
-e801322e946c1b81782a2d6b962be15e  unaligned-preprint-seqs.fasta.gz
+qiime genome-sampler sample-diversity --i-context-seqs filtered-context-seqs.qza --p-percent-id 0.9995 --o-selection diversity-selection.qza
 ```
 
-Brendan identified sequences to exclude based on low quality alignments or
-missing metadata. He also requested some specific sequences be added. These
-additions and subtractions result in the data used for our analyses.
+Last, we'll sample near neighbors of the focal sequences from the context sequences. Of the near-neighbor sequences that are identified for each focal sequence (up to 10 by default), 3 sequences will be selected at random such that each `location` (defined in the `context-metadata.tsv` file) has an equal probability of being selected for inclusion in downstream analysis.
+
+```
+qiime genome-sampler sample-neighbors --i-focal-seqs filtered-focal-seqs.qza --i-context-seqs filtered-context-seqs.qza --m-locale-file context-metadata.tsv --m-locale-column location --p-percent-id 0.9999 --p-samples-per-cluster 3 --o-selection neighbor-selection.qza
 ```
-$ az-covid-1/clean-seqs.py --remove brendans-ids-to-drop.tsv --fix-date True unaligned-preprint-seqs.fasta unaligned-cleaned-preprint-seqs-2020-04-28.fasta
-54 sequences were duplicated, 37 were removed, and 71 were renamed out of 466 total.
 
-$ grep -c '^>' unaligned-cleaned-preprint-seqs-2020-04-28.fasta
-375
+Now, we'll combine the results of the three sampling approaches and generate a summary of the full selection process.
 
-$ grep -A 1 EPI_ISL_407215 gisaid-unaligned.fasta >> unaligned-cleaned-preprint-seqs-2020-04-28.fasta
+```
+qiime genome-sampler combine-selections --i-selections date-selection.qza --i-selections diversity-selection.qza --i-selections neighbor-selection.qza --o-combined-selection master-selection.qza
```

Comment:
repeating the parameter flag isn't required anymore, so this would work:
```
--i-selections date-selection.qza diversity-selection.qza neighbor-selection.qza
```
which I think is prettier

---

or even:
```
--i-selections *-selection.qza
```